src/customers/account-dashboard-my-account.md

```diff
@@ -6,3 +6,9 @@ The My Account section of the customer dashboard gives customers an overview of
 
 ![]({% link images/images-ee/customer-account-dashboard-my-account.png %}){: .zoom}
 _My Account_{:.ee-only}
+
+## Recently Ordered Products
+
+The **Recent Orders** block is displayed in the sidebar and on the **My Account page**, this feature is available only for customers who are logged in. The **Recently ordered** block appears after placing an order, it displays 5 products from the last purchase.
+
+The customer can re-add products to the cart by checking the products and clicking on the **Add to Cart** button. You can also view the last order by clicking the **View all** button, the customer will be redirected to the **My account** page, to the **Recent Orders** block.
```

Comment:
Let's keep whole thing in third person. Also some edits for grammar and style

```suggestion
The customer can re-add products to the cart by selecting the products and clicking **Add to Cart**. They can also view the last order by clicking **View all**, which redirects to the _My account_ page to includes the **Recent Orders** block.
```

Reply:
@leslietilling thanks, done.